.github/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish to npm
```

Comment:
You have to do som GH action development to get this to work, it isn't going to work like this, not sure what the intent is here by including both these file, it was suppose to be an examples of some places we are doing pushes to package cloud to redevelop a workflow to push the express gateway package

Reply:
semantic-release has a plugin that can push the npm package too:  https://github.com/semantic-release/npm
Plus, if this is OSS, why don't we just push to a public npm repo.  @ontehfritz

Reply:
Because we need to see if it works with the image we have first, it hasn't been tested yet with ratehub-gateway

Reply:
yeah its better to first test with `ratehub-gateway` build then, I believe we can use [npm package](https://github.com/semantic-release/npm) to push it to public npm.

Reply:
yeah you can after we verify it works :)